src/evaluation/Run.jl

```diff
@@ -80,6 +80,9 @@ function run_reactive!(session::ServerSession, notebook::Notebook, old_topology:
 	delete!.([notebook.bonds], to_delete_vars)
 
 	local any_interrupted = false
+	ordered_cells = get_ordered_cells(new_order)
+	set_dependencies!(notebook, ordered_cells)
```

Comment:
The order of _any_ cell can change when the reactivity graph changes, not just that of cell that will rerun. For example: in a notebook with ten cells, each with code `123`, the cells will execute in oder 1, 2, ..., 10. But if you delete the first cell, the last cell changes its order to 9. The last cell is not a depender of the first cell, so it does not appear in `to_run`.

See my next comment :))))))))))

Reply:
In `updated_topology`, all information is available to calculate / set the depenencies (the TopologicalOrder needs to be recalculated).
The cell order (as `Vector{UUID}`) is now stored in `notebook`, this information could be re-used when saving the notebook.
In general I think I need better ways to test the dependencies inside a notebook - maybe some JS inside a Pluto cell could do this.

Reply:
I did the changes and it seems to work for me locally.
But unfortunately the Frontend tests fail, I do not know why. Do you have any idea?

Reply:
@pankgeorg will know